cmd/drone-agent/main.go

```diff
@@ -80,6 +80,16 @@ func main() {
 			Name:   "healthcheck",
 			Usage:  "enables the healthcheck endpoint",
 		},
+		cli.StringFlag{
```

Comment:
you can use `DurationFlag` here and then remove the `time.ParseDuration`

Reply:
Will implement this